File: backend/java/localpos-user-service/src/main/java/com/pds/localpos/userservice/dto/UserResponseDTO.java
package com.pds.localpos.userservice.dto;

import java.time.Instant;
import java.util.Set;

public record UserResponseDTO(
        String id,
        String username,
        String email,
        String firstName,
        String lastName,
        String phone,
        String address,
        boolean isActive,
        Set<StoreDTO> stores,
        Set<RoleDTO> roles,
        Instant createdAt,
        Instant updatedAt
) {
}

File: backend/kotlin/localpos-user-service-kt/src/main/kotlin/com/pds/localpos/userservice/dto/UserResponseDTO.kt
package com.pds.localpos.userservice.dto

import java.time.Instant

data class UserResponseDTO(
    val id: String,
    val username: String,
    val email: String?,
    val firstName: String?,
    val lastName: String?,
    val phone: String?,
    val address: String?,
    val isActive: Boolean,
    val stores: Set<StoreDTO> = emptySet(),
    val roles: Set<RoleDTO> = emptySet(),
    val createdAt: Instant,
    val updatedAt: Instant
)
